utilities/app/src/main/java/utilities/FizzBuzzTree.java

```diff
@@ -0,0 +1,39 @@
+package utilities;
+
+
+
+import utilities.tree.Node;
+import utilities.tree.Tree;
+
+public class FizzBuzzTree {
+
+
+    Tree newTree = new Tree();
+    public static void FizzBuzzTree(Tree tree) throws Exception {
+        _FizzBuzzTree(tree.root);
+    }
+    public static void _FizzBuzzTree(Node node) throws Exception {
+        if(node == null) throw new Exception("The tree is empty");
+        String currentAsAString = node.value.toString();
```

Comment:
Why do you take the value, turn it into a string, then parse it back into an integer?